src/components/Board.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState, useCallback } from "react";
+import { Square } from "./Square";
+import "../css/board.css";
+
+interface SquareInterface {
+    pieceColor: string,
+    color: string,
+    piece: string
+}
+
+export function Board() {
+    let [squares, setSquares] = useState<Array<Array<SquareInterface>>>();
+    let [firstRowOrder] = useState([
+        "rook",
+        "knight",
+        "bishop",
+        "queen",
+        "king",
+        "bishop",
+        "knight",
+        "rook",
+    ]);
+
+    const getPieceColor = (row: number) => {
+        if (row < 2)  return "black";
+        else if (row > 5) return "white";
+
+        return "";
+    };
+
+    const getPiece = useCallback((row, col) => {

```

Comment:
This doesn't actually need to be a callback, and can also be moved outside of the function.